src/components/Questionnaire/QuestionnaireResults/Table/TableILS.tsx

```diff
@@ -0,0 +1,174 @@
+    inter.set(-3, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(3, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-5, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(5, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-7, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(7, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-9, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(9, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(-11, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(11, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+  } else {
+    inter.set(-1, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(1, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-3, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(3, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-5, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(5, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-7, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(7, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-9, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(9, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(-11, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(11, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+  }
+
+  //if the interpretation is "balanced", then only return "balanced" without the Dimension
+  if (inter.get(score) === t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    return t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced')
+  return inter.get(score) + ' ' + interpretationString
+}
+
+//Depending on the score, return the corresponding dimension
+export const ILSDimension = (dimensionNumber: number, score: number, onlyEnglish?: boolean): string => {
+  const { t, i18n } = useTranslation()
+
+  switch (dimensionNumber) {
+    case 1:
+      if (onlyEnglish) {
+        const en = i18n.getFixedT('en')
+        if (score > 0) return en('components.Questionnaire.QuestionnaireResults.Table.TableILS.Active')
+        else return en('components.Questionnaire.QuestionnaireResults.Table.TableILS.Reflective')
+      } else {
```

Comment:
Hier else if statt neuem if. Mei mir meckert sonar cloud ^^
```suggestion
      } else 
```

Reply:
Das überall darunter auch